docs/02_installation.md

```diff
@@ -78,13 +85,13 @@ $ sudo rm /usr/local/bin/eosio-*
 * eosio-objdump
 * eosio-readelf
 
-Below tools are not installed after brew install, you get them only by building the repository and installing from scracth, [see here](#guided_installation_or_building_from_scratch)
-eosio-abidiff
-eosio-ranlib
-eosio-ar
-eosio-objdump
-eosio-readelf
+The following tools are not installed after brew install, you get them when build the repository and install it from scratch, [see here](#guided-installation-or-building-from-scratch):
```

Comment:
Tiny suggestion:

The following tools are not installed after brew install; you get them when you build the repository and install it from scratch, [see here](#guided-installation-or-building-from-scratch):

Reply:
thank you
updated